pkg/storage/db.go

```diff
@@ -320,11 +320,17 @@ func (db *DB) GetPartitions(ctx context.Context) ([]domain.Partition, error) {
 			End:       end,
 		})
 	}
-
 	if err := rows.Close(); err != nil {
 		return nil, err
 	}
-
+	for _, v := range partitions {
+		row := db.sqldb.QueryRow("SELECT count(*) FROM " + v.Name)
```

Comment:
task: This needs to use the [`QueryRowContext`](https://golang.org/pkg/database/sql/#Conn.QueryRowContext) method similar to how the other query above uses `QueryContext`. Passing the context variable around makes sure that our system behaves correctly if we decide to stop doing work. For example, if someone makes the API call to list all partitions but then closes their connection half-way through processing then we don't want to continue making queries to the database. The context is how we signal that a request is cancelled and work should stop.

Reply:
Changed to QueryRowContext. Thank you